inc/class-core-sitemaps-provider.php

```diff
@@ -295,9 +296,12 @@ public function calculate_sitemap_lastmod( $type, $subtype, $page ) {
 
 		$times = wp_list_pluck( $list, 'lastmod' );
 
-		usort( $times, function( $a, $b ) {
-			return strtotime( $b ) - strtotime( $a );
-		} );
+		usort(
+			$times,
+			function( $a, $b ) {
```

Comment:
This closure can be declared as static to improve preformance.

Reply:
I've simplified this using `wp_list_sort()` in e9b24b4.